os/net/lwip/src/api/sockets.c

```diff
@@ -453,31 +468,36 @@ struct lwip_sock *trycheck_selwait_socket(int s)
  */
 int alloc_socket(struct netconn *newconn, int accepted)
 {
-	int i;
+	int idx;
+	struct socketlist *list;
+	struct lwip_sock *sock;
+
+	list = sched_getsockets();
+	if (!list) {
+		ndbg("could not get the socketlist\n");
```

Comment:
It looks like LWIP_DEBUGF is used in most places in this file. Why are you using ndbg? Can we make it uniform?

Reply:
done